sw/host/tests/chip/gpio/src/sleep_pin_mio_dio_val.rs

```diff
@@ -0,0 +1,252 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{ensure, Result};
+use clap::Parser;
+use std::time::Duration;
+
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::execute_test;
+use opentitanlib::io::gpio::PinMode;
+use opentitanlib::io::gpio::PullMode;
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::uart::console::UartConsole;
+
+const MIO_PADS: &[&str] = &[
```

Comment:
Please could you extract these to `sw/host/opentitanlib/src/test_utils`? They will be useful for a test I'm writing.